byterun/win32.c

```diff
@@ -324,7 +324,7 @@ static void store_argument(char * arg)
 {
   if (argc + 1 >= argvsize) {
     argvsize *= 2;
-    argv = (char **) realloc(argv, argvsize * sizeof(char *));
+    argv = (char **) caml_stat_resize_noexc(argv, argvsize * sizeof(char *));
     if (argv == NULL) out_of_memory();
```

Comment:
You could call `caml_stat_resize` and get rid of the NULL test. Or do you think it's better to do such cleanup in a separate PR?